.gitlab/deploy_containers/conditions.yml

```diff
@@ -28,6 +28,30 @@
       DSD_REPOSITORY: dogstatsd
       IMG_REGISTRIES: public
 
+.manual_on_deploy_auto_on_rc-ot:
+  - if: $BUCKET_BRANCH != "beta" && $BUCKET_BRANCH != "stable"
+    when: manual
+    allow_failure: true
+    variables:
+      AGENT_REPOSITORY: agent-dev
+      IMG_REGISTRIES: dev
+  - if: $CI_COMMIT_TAG =~ /^[0-9]+\.[0-9]+\.[0-9]+-v[0-9]+\.[0-9]+\.[0-9]+-rc\.[0-9]+$/ && $FORCE_MANUAL != "true"
+    when: on_success
+    variables:
```

Comment:
🥜 If these are the default values we can maybe set them in the main `variables` section of the `.gitlab-ci.yml` (can maybe be a separate PR on our side as it would change a lot of existing conditions)

Reply:
These are all temporary jobs that will be removed once GA is reached for the OTel offering, so not sure it's worth it.

Reply:
Yes like the other comment, this is maybe something we could change to simplify the gitlab configuration and it's not directly related to your change so you can ignore this